action.yml

```diff
@@ -96,9 +96,9 @@ outputs:
     description: Some issues skipped when already transitioned or other causes.
     value: ${{ steps.transition.outputs.has_skippedIssues }}
     
-  some-not-found:
-    description: Some issues were not found in Jira.
-    value: ${{ steps.transition.outputs.has_notfoundIssues }}
+  some-excluded:
```

Comment:
If an issue is actual not found, it will error out instead of all us to get to the point where we can output results.  So instead, I am indicating here what issues are excluded or not found in the query that were passed into the `issues` input.